src/Microsoft.AspNetCore.Http/HttpContextFactory.cs

```diff
@@ -1,30 +1,51 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
+using System.Text;
 using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.Http.Features.Internal;
+using Microsoft.Extensions.ObjectPool;
 
 namespace Microsoft.AspNetCore.Http.Internal
```

Comment:
Yup. I have a second wave of breaking changes (after moving the feature implementations) queued up. That'll cover both #549 and #592.